src/Bootstrap/Tab.elm

```diff
@@ -366,6 +367,39 @@ view state ((Config { items }) as config) =
                     )
                 ]
 
+{-| Creates a tab control which keeps track of the selected tab item and displays the corresponding
+tab pane for you, plus performs an action when a tab is selected
+
+    Tab.viewWithAction model.tabState
+        <| Tab.items
+            [ Tab.item
+                { id = "tabItem1"
+                , link = Tab.link [] [ text "Tab 1" ]
+                , pane = Tab.pane [] [ text "Tab 1 Content" ]
+                }
+            , Tab.item
+                { id = "tabItem2"
+                , link = Tab.link [] [ text "Tab 2" ]
+                , pane = Tab.pane [] [ text "Tab 2 Content" ]
+                }
+            ]
+        <| Tab.config TabMsg
+        <| loadTabContent -- an action to perform when a tab is selected
+
+
+    loadTabContent : String -> Msg -> Msg
+    loadTabContent tagURL _ =
+        Msg.FetchContent tagURL
+-}
+viewWithAction : State -> Config msg -> (String -> msg -> msg) -> Html.Html msg
+viewWithAction state ((Config { items }) as config) action =
+    case getActiveItem state config of
```

Comment:
not a big fan of calling the same function twice, here and in view, but i guess its a quick win ;)